src/lib/Scenarios.ts

```diff
@@ -0,0 +1,16 @@
+import { WorldSetup } from "./SimulatorState";
+
+const GDP_US = 2e13; // 20 trillion dollars: annual US GDP
```

Comment:
This is the setup for the US scenario. I might rename __WorldSetup__ to __Scenario__ at some point in the future.